docs/resources/Channel.md

```diff
@@ -15,14 +15,14 @@ Represents a guild or DM channel within Discord.
 | permission\_overwrites? | array of [overwrite](#DOCS_CHANNEL/overwrite-object) objects | explicit permission overwrites for members and roles  |
 | name? | string | the name of the channel (2-100 characters) |
 | topic? | string | the channel topic (0-1024 characters) |
-| last\_message\_id? | snowflake | the id of the last message sent in this channel (may not point to an existing or valid message) |
```

Comment:
The questionmarks right after `last_message_id` and `parent_id` indicates that the field is nullable

Reply:
No, it indicates that they are optional. 
https://discordapp.com/developers/docs/reference#optional-resource-fields-example-nullable-and-optional-field

Reply:
https://discordapp.com/developers/docs/reference#optional-resource-fields-example-nullable-and-optional-field

>**Nullable Resource Fields**
>Resource fields that may be null have __types that are prefixed with a question mark.__
>**Optional Resource Fields**
>Resources fields that are optional have __names that are suffixed with a question mark.__

Question marks after field names mean that this is an optional field, but not nullable.

Reply:
Oh well, my bad then :)